packages/@n8n/api-types/src/dto/api-keys/create-or-update-api-key-request.dto.ts

```diff
@@ -8,6 +8,16 @@ const xssCheck = (value: string) =>
 		whiteList: {},
 	});
 
+const expirationDateCheck = (value?: number) => {
```

Comment:
This checks that the expiration is in the future, right? Maybe a better name could be `isTimeNotDefinedOrInFuture`